common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -797,6 +801,15 @@ private static boolean isAndroid0() {
         return android;
     }
 
+    private static boolean explicitNoReflectiveAccessible0() {
+        // we disable reflective access
+        return SystemPropertyUtil.getBoolean("io.netty.noReflectiveAccessible", javaVersion() >= 9);
```

Comment:
can we avoid the negative "no" in the name? IMHO it makes it more difficult to have to negate the logic to understand what is going on. What about `io.netty.tryReflectionSetAccessible`?

Reply:
@Scottmitch sure... I just used it because we use it for unsafe as well